src/lib/components/PostPreview/direction-properties.ts

```diff
@@ -0,0 +1,23 @@
+import { DirectionEnum } from '../../types';
+
+export type IDirectionPropertiesInterface = {
```

Comment:
`export type IDirectionPropertiesInterface `

if you want to use `interface` use `interface` instead of `type`
`IDirectionProperties` - `I` means `interface` at the start, no need to have `interface` at the variable name

Reply:
fixed